README.md

```diff
@@ -2,86 +2,78 @@
+  
+  Example users: [MixRank](https://www.citusdata.com/customers/mixrank), [Windows team](https://techcommunity.microsoft.com/t5/azure-database-for-postgresql/architecting-petabyte-scale-analytics-by-scaling-out-postgres-on/ba-p/969685)
+
+- **[Software-as-a-service (SaaS) applications](http://docs.citusdata.com/en/latest/use_cases/multi_tenant.html)**:
+  SaaS and other multi-tenant applications need to be able to scale their database as the number of tenants/customers grows. Citus enables you to transparently shard a complex, data model by the tenant dimension such to enable your database to grow along with your business.
+  
+  By distributing tables along a tenant ID column and co-locating data for the same tenant, Citus can horizontally scale complex (tenant-scoped) queries, transactions, and foreign key graphs. Reference tables and distributed DDL commands make database management a breeze compared to manual sharding. On top of that, you have a built-in distributed query engine for doing cross-tenant analytics inside the database.
+  
+  Example users: [Copper](https://www.citusdata.com/customers/copper), [Salesloft](https://fivetran.com/case-studies/replicating-sharded-databases-a-case-study-of-salesloft-citus-data-and-fivetran),
 
 ### Getting started with Citus
 
-The fastest way to get up and running is to deploy Citus in the cloud. You can also setup a local Citus database cluster with Docker.
+The fastest way to get started with Citus is to use the [Hyperscale (Citus)](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-hyperscale-portal) deployment option in Azure Database for PostgreSQL or [set up Citus locally](http://docs.citusdata.com/en/latest/installation/single_machine.html).
 
 #### Hyperscale (Citus) on Azure Database for PostgreSQL
 
-Hyperscale (Citus) is a deployment option on Azure Database for PostgreSQL, a fully-managed database as a service. Hyperscale (Citus) employs the Citus open source extension so you can scale out across multiple nodes. To get started with Hyperscale (Citus), [learn more](https://www.citusdata.com/product/hyperscale-citus/) on the Citus website or use the [Hyperscale (Citus) Quickstart](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-hyperscale-portal) in the Azure docs.
+You can get a fully-managed Citus cluster in minutes through the Hyperscale (Citus) deployment option in the [Azure Database for PostgreSQL](https://azure.microsoft.com/en-us/services/postgresql/) portal. Azure will manage your backups, high availability through auto-failover, software updates, monitoring, and more for all of your servers. To get started with Hyperscale (Citus), use the [Hyperscale (Citus) Quickstart](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-hyperscale-portal) in the Azure docs.
 
-#### Citus Cloud
+### Usage example: Distributed tables
```

Comment:
Would it make sense to give an example for reference table as well?

I kind of wonder what would happen if we have some more examples here, I know that there is a documentation link here but some people might not be clicking that, yet if we had more examples here, they might take a look at it. 

Can we include some benchmark numbers in the README such as from Hammerdb? (Maybe a chart that shows the horizontal scalability of citus with different number of nodes)

Could we have a chart here that shows the supported major PG version for each version? For example for citus 9.5 (pg11, pg12, pg13)